tests/components/zha/test_climate.py

```diff
@@ -1458,7 +1458,8 @@ async def test_set_moes_operation_mode(
     [
         (0, PRESET_AWAY),
         (1, PRESET_SCHEDULE),
-        # (2, PRESET_NONE),  # TODO: why does this not work?
+        # pylint: disable-next=fixme
+        (2, PRESET_NONE),  # TODO: why does this not work?
```

Comment:
```suggestion
        # (2, PRESET_NONE),  # TODO: why does this not work?
```

Was previously comment out